deployment/aks-periscope.yaml

```diff
@@ -148,7 +151,7 @@ metadata:
   name: collectors-config
   namespace: aks-periscope
 data:
-  COLLECTOR_LIST: managedCluster # <custom flag>
+  COLLECTOR_LIST: managedCluster, OSM, SMI # <custom flag>
```

Comment:
Should I reverse this?

Reply:
I think we had decided that by default these collectors will not be run, but can be enabled by users optionally. @Tatsinnit should we be adding documentation around this?

Reply:
Resolved in d34fbb85991d6fa1822bcf29d4b0158c5b3af79d